### DISCUSSION TITLE
What username format should be used for the 'Default follows for new users' in Settings?

### DISCUSSION THREAD

**Discussion by jsailo** [mastodon/mastodon#19432] [General]
Hello all - For **Default follows for new users** and **Recommend these accounts to new users** setting, should it be:

1. `@username`
OR
2. Just `username`
OR
3. `@username@mydomain.net`

I'm currently using `@username` with a `.` (comma) separating the usernames. Is this correct?

![image](https://user-images.githubusercontent.com/35369823/197466365-1405df18-cc21-4f62-bfe9-de4cfaf2e73a.png)

**Comment by Gargron** [comment]
There is validation on that field, if the system didn’t understand your input it would complain. All of these work.

**Comment by supernovae** [comment]
Has default follows been removed in 4.0? I'm not seeing it. Only recommended accounts.
